crates/task-impls/src/transactions.rs

```diff
@@ -29,6 +31,18 @@ use crate::{
     helpers::broadcast_event,
 };
 
+/// Builder Provided Responses
+pub struct BuilderResponses<TYPES: NodeType> {
```

Comment:
I'm not clear on why we're bundling this all in one struct. Are we collecting what we get from each phase with the builder?

Reply:
Yes, we are bundling all the responses from the builder. Later, we might filter and store what we need.

Reply:
Doesn't that defeat the point of, well, having an external builder?

Reply:
Sorry, I didn't get wdym?

Reply:
Never mind, we can fix this during the transition to multi-builder auctions